Makefile

```diff
@@ -1,67 +1,62 @@
 # Always keep the future version here, so we won't overwrite latest released manifests
+OPERATOR_SDK_VERSION ?= v0.19.2
+OPERATOD_SDK_CSV_VERSION ?= v0.8.0
 
-OPERATOR_SDK_VERSION ?= v0.8.0
 IMAGE_REGISTRY ?= quay.io/fromani
 IMAGE_TAG ?= latest
 OPERATOR_IMAGE ?= kubevirt-ssp-operator-container
 REGISTRY_IMAGE ?= kubevirt-ssp-operator-registry
 
-container-build: csv-generator container-build-operator container-build-registry
-
-container-build-operator:
+container-build: generate csv-generator
 	docker build -f build/Dockerfile -t $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG) .
 
-container-build-registry:
-	docker build -f build/Dockerfile.registry -t $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG) .
-
-container-push: container-push-operator container-push-registry
-
-container-push-operator:
+container-push:
 	docker push $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG)
 
-container-push-registry:
-	docker push $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG)
-
-container-release:
-	./hack/docker-push.sh $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG) $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG)
-
-csv-generator: operator-sdk
+csv-generator: operator-sdk-csv
 	./build/make-csv-generator.sh
 
 operator-sdk:
 	curl -JL https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_VERSION)/operator-sdk-$(OPERATOR_SDK_VERSION)-x86_64-linux-gnu -o operator-sdk
 	chmod 0755 operator-sdk
 
-operator-courier:
-	pip3 install --user operator-courier
+operator-sdk-csv:
+	curl -JL https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOD_SDK_CSV_VERSION)/operator-sdk-$(OPERATOD_SDK_CSV_VERSION)-x86_64-linux-gnu -o operator-sdk-csv
```

Comment:
is this a valid design to have csv version different from the sdk? I'm now aware if it can cause any downsides

Reply:
I did not see any downsides, the good thing is that `operator-sdk` commands are very specific. When you generate a CSV, it only reads the manifests from `deploy/` and does not generate them, so if you use one version to generate CSVs and another to generate CRDs, we should be fine